_posts/research/2020-04-22-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Autonomous Coordination of Cooperative Multi-Robot Systems
+description: Multi-Agent Reinforcement Learning for Robot Control
+author: Joewie J. Koh and Guohui Ding
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 75%
+excerpt_separator: <!-- More -->
+---
+
+Heterogeneous teams of cooperative multi-robot systems have the potential to propel robotics at large toward a safer, more accessible, and lower cost future.
+Through distribution of roles among the various agents, a multi-robot system can be imbued with capabilities that are more than the sum of its parts by coordinating multiple agents to cooperatively accomplish common goals; using search and rescue as an example, cooperation between unmanned aerial and ground vehicles could increase the odds of success with larger areas covered, better mobility.
+However, autonomous coordination of robots cannot rely on predefined rules or simple control schemes.
+Robots are both generators and consumers of data, and appropriate decision-making models that map observed data to actions can endow a multi-robot system with intelligent behaviors that improve cooperation.
+Our work proposes several approaches to fully leverage multi-agent RL for autonomous robot coordination.
+
+<!-- More -->
+
+{% include image.html url="research/multirobot/cooperative_manipulation.png" max-width="75%" description="An object too heavy or unwieldy for a single robotic arm could be manageable for two cooperating robots. Robots cooperating as needed could increase the efficiency of robotic systems, as the robots can work on independent tasks at other times." %}
+
+# Challenges
+
+One of the main challenges in intelligent robotics is interaction with the environment to achieve a goal, which motivates the question of "how a robot should learn to manipulate the world around it" [1].
```

Comment:
interaction with the environment to achieve a goal " is not strong enough. Also, move this to a positive denotation: don't talk about challenges but talk about vision :-)